src/main.js

```diff
@@ -38,32 +41,52 @@ let App = (
         {/* Routes */}
         <div>
           {/* Matches /disclaimer */}
-          <Fragment forRoute="/disclaimer"><MainDisclaimerPage/></Fragment>
+          <Fragment forRoute="/disclaimer">
+            <RichLayout><MainDisclaimerPage/></RichLayout>
+          </Fragment>
           {/* Matches / */}
-          <Fragment forRoute="/"><LandingPage/></Fragment>
+          <Fragment forRoute="/">
+            <Layout><LandingPage/></Layout>
+          </Fragment>
 
           {/* Matches /augsburg/de */}
           <Fragment forRoute="/:location/:language">
             <div>
               {/* Matches /augsburg/de/search -> Search */}
-              <Fragment forRoute="/search"><SearchPage/></Fragment>
+              <Fragment forRoute="/search">
+                <RichLayout><SearchPage/></RichLayout>
+              </Fragment>
               {/* Matches /augsburg/de/disclaimer -> Disclaimer */}
-              <Fragment forRoute="/disclaimer"><DisclaimerPage/></Fragment>
+              <Fragment forRoute="/disclaimer">
+                <RichLayout><DisclaimerPage/></RichLayout>
+              </Fragment>
               {/* Matches /augsburg/de/events -> Events */}
-              <Fragment forRoute="/events"><EventsPage/></Fragment>
-              {/* Matches /augsburg/de/redirect/ -> Redirect */}
-              <Fragment forRoute="/redirect"><PageRedirector/></Fragment>
+              <Fragment forRoute="/events">
+                <RichLayout><EventsPage/></RichLayout>
+              </Fragment>
+              {/* Matches /augsburg/de/redirect -> Redirect */}
+              <Fragment forRoute="/redirect">
+                <Layout><PageRedirector/></Layout>
+              </Fragment>
+              {/* Matches /augsburg/de/fetchPdf/* -> Redirect */}
+              <Fragment forRoute="/fetchPdf/*">
```

Comment:
I think we should avoid having camelCase in the url. `fetch-pdf` would be more appropriate imo

Reply:
why did you change from RichLayout to a normal Layout for PageRedirector and PdfFetcherPage? I think it's flickering with the normal layout which doesn't have our header

Reply:
Yup, pageredirector should have Richlayout, whereas PDF Fetcher shouldnt have a Richlayout (since its Always opened in a New Tab where we just wait for the PDF to have been generated and immediately redirect to that PDF. So there's no header necessary there.)

Reply:
But we should disable the buttons in richlayout. I'm not sure what happens if you change the language during a redirect :D